sdk/storage/Azure.Storage.Blobs/src/Sas/KeySasQueryParametersExtensions.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace Azure.Storage.Sas
+{
+    internal static class KeySasQueryParametersExtensions
```

Comment:
Nit - this name is kind of odd.   "Key" is a pretty generic type name.  I'd like it to either be `DelegationKey...` or maybe move these into `SasExtensions`.

Reply:
I separated these because they only apply to Blobs/Datalake whereas SasExtensions apply to all libraries. I can rename it.